style.css

```diff
@@ -696,53 +707,83 @@ body {
 
 @media (max-width: 768px) {
   #contact {
+    background-color: #fff;
     width: 375px;
   }
 
-  .contact-container {
-    background-image: url(image/contact-form-shapes.png);
-  }
-
-  .contact-me div {
-    display: flex;
-    justify-content: center;
+  #contact .contact-case {
+    background-color: #6070ff;
+    padding: 0 0;
   }
 
-  .contact-container .contact-me {
+  .contact-image {
     display: flex;
     justify-content: center;
+    background-image: url(image/contact-form-shapes.png);
+    background-color: #6070ff;
+    background-repeat: no-repeat;
+    background-size: cover;
+    padding: 150px 0;
+    border-radius: 100px 0 0 0;
   }
 
-  .contact-container .para {
+  .contact-container h2 {
     display: flex;
     justify-content: center;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #fff;
+    margin-bottom: 20px;
   }
 
-  .contact-container .para p {
+  .contact-container p {
     width: 327px;
     text-align: center;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #ebebff;
+    margin-bottom: 20px;
   }
```

Comment:
At certain breakpoints of the desktop version, certain parts of the page at the left side is cut off as shown in the images below. Kindly check and fix that. 


![Screenshot from 2023-01-27 19-14-45](https://user-images.githubusercontent.com/65233860/215179410-9206ebf0-60ae-4e86-975e-002951ff394d.png)


![Screenshot from 2023-01-27 19-15-05](https://user-images.githubusercontent.com/65233860/215179440-a645a999-5975-41ff-addd-b98062e80f34.png)


![Screenshot from 2023-01-27 19-27-32](https://user-images.githubusercontent.com/65233860/215179602-c0cc6c9c-5497-4b86-b46b-9136a8342f23.png)